flow/record/fieldtypes/__init__.py

```diff
@@ -15,9 +15,15 @@
 from urllib.parse import urlparse
 
 try:
-    from zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+    try:
+        from zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+    except ImportError:
+        from backports.zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+    HAS_ZONE_INFO = True
 except ImportError:
-    from backports.zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+    warnings.waring("Could not use zoneinfo, using the default timezone 'UTC'.")
```

Comment:
```suggestion
    warnings.warn("Could not use zoneinfo, using the default timezone 'UTC'.")
```

Reply:
typo here, but I would remove the warning here all together, and move it to my suggestion below.
I think raising the warning only if FLOW_RECORD_TZ has been set is the more user friendly.